Cloud_Providers.md

```diff
@@ -0,0 +1,50 @@
+solution is right for your environment, you must consider the monitoring needs of both the cluster administrator and the SAS Viya administrator and compare the capabilities of the monitoring solutions. See [Choose a Monitoring Strategy for an Azure Environment](https://go.documentation.sas.com/?cdcId=sasadmincdc&cdcVersion=v_001LTS&docsetId=calmonitoring&docsetTarget=n0okndfnjiwcxmn1ci29kjroaamp.htm&locale=en#n006zgl0bnp8txn15azzj13z8ww8) in the SAS Viya documentation for complete information. 
+
+## Amazon Elastic Kubernetes Service (EKS)
+
+### Deploying SAS Viya Logging and Monitoring on Amazon EKS
+
+In order to use the SAS Viya monitoring and logging components in an Amazon EKS environment, you must deploy monitoring and logging using ingress. See the [Ingress sample](/samples/ingress/README.md) for deployment information.
+
+No other customization is needed to deploy and run the SAS Viya logging and monitoring components on Amazon EKS.
+
+### Using Amazon EKS Logging and Monitoring Applications With SAS Viya
+
+If you deploy SAS Viya in an Amazon EKS environment, you can view metric information by using either the SAS Viya monitoring components or Amazon's monitoring application, CloudWatch. In order to use CloudWatch to monitor SAS Viya metrics, you must deploy the CloudWatch agent for Prometheus, make configuration changes to convert the Prometheus metrics collected from SAS Viya into performance log events, and map the log events to CloudWatch metrics. Follow the steps in the [Amazon CloudWatch Integration sample](/samples/cloudwatch/README.md) to use CloudWatch to monitor a selected group of SAS Viya metrics. Not all SAS Viya metrics are supported by this sample, because supporting all SAS Viya metrics would require every possible combination of labels from every SAS Viya source and every metric to be mapped to CloudWatch metric types. This sample might be updated in later releases to include additional combinations of dimensions for key metrics.
+
+Amazon CloudWatch can also display log messages collected from Fluent Bit, which is the application used by the SAS Viya logging solution. However, current samples do not provide the configuration needed to integrate SAS Viya log processing with CloudWatch.  
+
+## Google Kubernetes Engine (GKE)
+
+### Deploying SAS Viya Logging and Monitoring on GKE
+
+In order to use the SAS Viya monitoring and logging components in an GKE environment, you must deploy monitoring and logging using ingress. See the [Ingress sample](/samples/ingress/README.md) for deployment information.
+
+Depending on the size of your cluster, you might need to increase the default size of the virtual machines in your nodepool and to split the workload among multiple nodes in order for the logging and monitoring components to deploy and run successfully. 
```

Comment:
Add a subhead of "Tuning Specifics"? Or, "Deployment Considerations" or...

Reply:
I don't know that we need to say this anymore. The issue came up when the IAC deployment picked small VMs, but that has been addressed, so there's no action to take in most cases.

Reply:
Paragraph removed